spec/elastic_apm_spec.rb

```diff
@@ -14,7 +14,10 @@
   end
 
   context 'when running', :mock_intake do
-    before { ElasticAPM.start }
+    before do
+      # disable immediate metrics collection
+      ElasticAPM.start(metrics_interval: 0)
```

Comment:
@estolfo Not really a fan but it solves the JRuby test failures. The delegation tests are already a bit weird (although they have discovered errors before.)

_Wdyt?_